openshift/api/src/main/java/org/arquillian/cube/openshift/api/AddRoleToServiceAccounts.java

```diff
@@ -33,6 +33,6 @@
  */
 @Retention(RetentionPolicy.RUNTIME)
 @Target(ElementType.TYPE)
-public @interface AddRoleToServiceAccountWrapper {
+public @interface AddRoleToServiceAccounts {
```

Comment:
Hmm I see why they called it a `Wrapper` instead of using simple plural. Maybe call it `AddRolesToServiceAccounts`?

Reply:
so plural roles and plural accounts

Reply:
Right.